docs/index.rst

```diff
@@ -56,6 +56,9 @@ Contents
 :doc:`scylla-cloud-serverless`
     Connect to ScyllaDB Cloud Serverless
 
+:doc:`column-encryption`
```

Comment:
We should consider if we want to support this feature or not

Reply:
why not, it is completely client-side feature

Reply:
more so, i think it makes sense to have an epic to bring it into other drivers

Reply:
exactly that, it's not a feature we planned nor tested.

if we do plan and test, that's great, I would document it and "promate" it, before that.

Reply:
It is being tested at `tests/integration/standard/column_encryption/test_policies.py` and I don't think there anything that could trigger scylla-specific issue in this code, it encrypts original content, and serialize it to the blob and does reverse on unserialization.

Reply:
I didn't said there anything scylla related in it, or cassandra specific.

just that we did tested it, having integration tests is nice, but not enough for officially support this.
for example, do we know it's performence effect ?

just suggesting we should communicate with broader audience, on such commitments. (i.e. putting in the docs, makes it a commitment for it to be working, and for its quality)

Reply:
Sure, let's invoke @roydahan and @mykaul. For context:
We are syncing python-driver with the upstream and in 3.28.0 datastax [released](https://docs.datastax.com/en/developer/python-driver/3.27/column_encryption/index.html) client-isde column encryption    
.
Core idea of this feature is to seamlessly encrypt and decrypt a `blob` on client side.
Along side with this feature they have integration tests with it which we are running.
My proposal is to include this feature into our documentation right away, reasoning is:
1. It is entierly client-side feature, and chance of getting scylla-specific bug is close to 0.
2. It is tested at integration suite.
3. Performance degradation is expected and stays at users discretion.

Reply:
Yes, we've seen this feature.
Let's start with an Epic for "all drivers", there we can discuss and decide if we want it and if needed for all drivers.

Reply:
Issue is [here](https://github.com/scylladb/scylla-drivers/issues/15), so, should we remove it from the documentation ?

Reply:
Maybe we should write it as experimental for now.